app/src/main/java/org/greenstand/android/TreeTracker/database/dao/PlanterDao.kt

```diff
@@ -21,6 +22,13 @@ interface PlanterDao {
     @Query("SELECT * FROM planter_identifications WHERE photo_url IS NULL")
     fun getPlanterIdentificationsToUpload(): List<PlanterIdentificationsEntity>
 
+    @Transaction
+    @Query("SELECT * FROM planter_details WHERE identifier = :identifier")
+    fun getPlanterDetailsByIdentifier(identifier: String): LiveData<PlanterDetailsEntity>
```

Comment:
Using LiveData usually means that we are getting updates of some type of data, in this case PlanterDetailsEntity. I'm guessing we only need to load this once with no need for updates. If that's the case, we can just return a plain PlanterDetailsEntity

Reply:
I tried this approach but I have a bug and I had struggled with it for two days. I can't figure out what I'm doing wrong. You can see in my last commit what I've done. When you have some time maybe you could take a look. Thank you so much!

Reply:
If it doesn't crash the app we can probably merge what you have to allow additional people to figure out the bugs. Sound good @LaviniaDragunoi @deepwinter ?

Reply:
The last commit is crashing the app. I could change it to the old version of my  code that is not the best approach but is not crashing the app. So I think we should wait a little with the merge.

Reply:
Changing it to the old version and getting this merged sounds like a good idea.